.github/workflows/renovate.yml

```diff
@@ -40,6 +45,8 @@ env:
   # Override loglevel if set
   LOG_LEVEL: ${{ inputs.logLevel || 'info' }}
   RENOVATE_CONFIG_FILE: .github/renovate.json
+  # Override minimumReleaseAge if set
+  RENOVATE_MINIMUM_RELEASE_AGE: ${{ inputs.minimumReleaseAge != 'null' && inputs.minimumReleaseAge || '' }}
```

Comment:
We have not introduced changes on this workflow for a while as it has been deprecated in favour of the `-vault.yml` version. If users within the ecosystem needs this, they are likely better of moving on to the new workflow.